pandas-stubs/core/tools/timedeltas.pyi

```diff
@@ -31,7 +31,7 @@ def to_timedelta(
 @overload
 def to_timedelta(
     arg: (
-        Sequence[float | timedelta]
+        Sequence[str | float | timedelta]
         | list[str | float | timedelta]
```

Comment:
Combine these to one item of `Sequence[str | float | timedelta]`

Reply:
Indeed Sequence is encompassing list so might as well.